k8s/mongodb/mongo-ss.yaml

```diff
@@ -37,14 +37,30 @@ spec:
       terminationGracePeriodSeconds: 10
       containers:
       - name: mongodb
-        image: mongo:3.4.1
+        # TODO(FIXME): Do not use latest in production as it is harder to track
+        # versions during updates and rollbacks. Also, once fixed, change the
```

Comment:
This will change once we frontend mongo with nginx.
Actually, it has already been changed and being tested. :+1: